lib/que/web/sql.rb

```diff
@@ -77,6 +77,20 @@ def delete_jobs_query(scope)
     LIMIT $1::int
     OFFSET $2::int
   SQL
+  chi_data_events: <<-SQL.freeze,
+    SELECT *
+    FROM chi_events
+    WHERE
+      data @> $1::jsonb
+    ORDER BY event_order desc
```

Comment:
Rather than putting the limit in the app, let's pass it in here (and the next query) with 
```
LIMIT $2::int
```
That way postgresql does only as much as we need

Reply:
Given it's intended for human consumption, I'm thinking that somewhere between 500-2000 would be a reasonable number... I'd tend towards the low end of 500 personally.    This tools isn't intended to present all that much data.